src/order/bounded_lattice.lean

```diff
@@ -23,6 +23,8 @@ instances for `Prop` and `fun`.
 * `semilattice_<sup/inf>_<top/bot>`: Semilattice with a join/meet and a top/bottom element (all four
   combinations). Typical examples include `ℕ`.
 * `bounded_lattice α`: Lattice with a top and bottom element.
+* `distrib_lattice_bot α`: Bounded and distributive lattice. Typical examples include `Prop` and
+  `set α`.
```

Comment:
This comment isn't useful, as it doesn't actually provide an example beyond " all `bounded_distrib_lattice`s are `distrib_lattice_bot`s". Is the following true?
```suggestion
* `distrib_lattice_bot α`: Bounded and distributive lattice. An example is `finset` over an infinite type
```
If so, can you add that instance?

Reply:
Whoops, that's a bad copy-paste. It is true for `finset`, through `generalized_boolean_algebra`.

Reply:
In that case, the comment should probably say that it's a common base for `generalized_boolean_algebra` and `distrib_lattice_bot`